salt/containerd/map.jinja

```diff
@@ -0,0 +1,5 @@
+{%- set containerd = salt['grains.filter_by']({
+  'Linux': {
+    'version': 'latest'
```

Comment:
https://www.lemonde.fr/international/article/2019/02/07/la-france-rappelle-son-ambassadeur-en-italie-et-denonce-des-attaques-sans-precedent-depuis-la-fin-de-la-guerre_5420480_3210.html

Reply:
Eventually, yes.

Reply:
Yes agree the point is that by default we use latest but we can overwrite it in the pillar

Reply:
Yeah, I see the logic.

But the default value should be a valid one, and by using latest we take the (limited?) risk to pick "by default" an incompatible/untested version (different major version number).

Anyway, since we're at the start of the project nothing is freezed yet I guess, so we can go with latest for now.